src/Abc.Zebus/Core/Bus.cs

```diff
@@ -350,13 +335,16 @@ private void OnSubscriptionsUpdatedForTypes(HashSet<MessageTypeId> updatedTypes)
             _directory.UpdateSubscriptions(this, subscriptionUpdates);
         }
 
-        public void Reply(int errorCode)
+        public void Reply(int errorCode) => Reply(errorCode, null);
```

Comment:
I know it did not exist before and might break some usages, but it would be nice to check for 0/1 special error codes here.